README.md

```diff
@@ -343,7 +343,7 @@ In order to perform local development on this codebase:
 
 * Check out the repository: `git clone https://github.com/hasura/ndc-typescript-deno.git`
 * This assumes that you will be testing against function in `./functions`
-* Vendor the dependencies with `cd ./function && deno vendor -f index.ts`
+* Vendor the dependencies with `cd ./function && deno vendor --node-modules-dir -f index.ts`
```

Comment:
This is no longer required if they are using the preVendor config option which is true by default.

Reply:
This is under the "contribution to this connector" section, and further down the configuration they are told to pass does not specify preVendor, which means it's undefined, which is default false.

The "defaulting" to true that was done only does it in `update_configuration`, which simply adds an explicit `preVendor: true` to someone's config if they POST to the config server.

Reply:
I guess we should support the defaulting outside of update-configuration to be honest. And remove the explicit update.